### DISCUSSION TITLE
Making tags case-insensitive

### DISCUSSION THREAD

**Discussion by haroon-sheikh** [getgauge/gauge#2038] [General]
1. Would it be possible to make the tags case insensitive when filtering the specs/scenarios for execution? 
2. Or is there a reason why they are not?

What would involve in change these? Is this likely to break anything?

**Comment by haroon-sheikh** [comment]
Do we envisage any potential issues with this PR https://github.com/getgauge/gauge/pull/2039

**Reply by zabil** [reply]
Don't think it should be an issue or affect backward compatibility unless there are users that differentiate upper case and Lower case. For backwards compatibilty, I think it's worth making this an option with the default behaviour being case sensitive.

**Reply by haroon-sheikh** [reply]
Thanks @zabil. I've gone through all tests, and tested against all of our current repositories and not seen any issues. 👍
